src/tools/illink/src/linker/Linker.Dataflow/MethodBodyScanner.cs

```diff
@@ -288,7 +288,7 @@ protected virtual void Scan (MethodIL methodIL, ref InterproceduralState interpr
 
 			BasicBlockIterator blockIterator = new BasicBlockIterator (methodIL);
 
-			ReturnValue = new ();
+			MultiValue ReturnValue = default;
```

Comment:
Test are failing because by making the type MultiValue now ReturnValue is treated as a local variable instead of MethodBodyScanner.ReturnValue
```suggestion
			ReturnValue = default;
```